tests/test_disco3.py

```diff
@@ -12,5 +15,12 @@ def disco3():
 
 
 def test_parse_xml(disco3):
-    poem = parse_xml("tests/fixtures/input-disco-3.xml")
-    assert [*poem] == disco3
+    poem = parse_xml("tests/fixtures/input_disco3.xml")
```

Comment:
Try to avoid `tests/fixtures/disco.json` and use a proper Path() instead for win/unix compatibility

Reply:
Done